partcad-ide-vscode/bundled/tool/lsp_jsonrpc.py

```
@@ -58,7 +58,9 @@ def write(self, data):
         with self._lock:
             content = json.dumps(data)
             length = len(content.encode("utf-8"))
-            self._writer.write(f"{CONTENT_LENGTH}{length}\r\n\r\n{content}".encode("utf-8"))
+            self._writer.write(
+                f"{CONTENT_LENGTH}{length}\r\n\r\n{content}".encode("utf-8")
```

Ruff (0.8.2)

Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)